### DISCUSSION TITLE
Why VStack can't render children (I used typescript)

### DISCUSSION THREAD

**Discussion by auvansangit** [chakra-ui/chakra-ui#6980] [Q&A]
As the title, please see the sample bellow:

![image](https://user-images.githubusercontent.com/20507461/201621676-d20cbb57-6fe3-4edd-9ca6-79197431f14e.png)

My expectation that the VStack can render children props as Box or other Component!

**Comment by zgid123** [comment]
for my opinion, VStack or Stack or HStack has something called `spacing`, it will set `margin` for the children (for Stack, it is `margin-top`), so when you render the text only, it cannot set that for the text. So they reject the text and only render the component instead of raw text I guess
